src/double.js

```diff
@@ -0,0 +1,57 @@
+const ms = require('ms');
+
+export default class DoubleDecker {
+
+  constructor (one, two) {
+    this.one = one;
+    this.two = two;
+  }
+
+  get (k, cb) {
+    return this.one.get(k, (err, res) => {
+      if (err) return cb(err);
+      if (!res) return cb(null, null);
+      if (res < Date.now()) {
```

Comment:
Why do we not just chain this? 
`return this.del(this.one, this.two, k).then(function() { return this.two.get(k, cb); })`
Is it to not have to wait for the response? Maybe I'm missing something! :)